calculators/candidate_contributions_by_type.rb

```diff
@@ -72,9 +76,12 @@ def contributions_by_candidate_by_type
   def unitemized_contributions_by_candidate
     @_unitemized_contributions_by_candidate ||= {}.tap do |hash|
       results = ActiveRecord::Base.connection.execute <<-SQL
-        SELECT "Filer_ID", "Amount_A" FROM "efile_COAK_2016_Summary"
+        SELECT DISTINCT ON ("Filer_ID", "Amount_A")
+          "Filer_ID", "Amount_A" FROM "efile_COAK_2016_Summary"
         WHERE "Filer_ID" IN ('#{@candidates_by_filer_id.keys.join "','"}')
           AND "Form_Type" = 'A' AND "Line_Item" = '2'
+        GROUP BY "Filer_ID", "Amount_A", "Report_Num"
+        ORDER BY "Filer_ID", "Amount_A", "Report_Num" DESC
```

Comment:
Yeah, grouping by "Amount_A" is a bit weird perhaps, but since each F460 has exactly one row matching the WHERE, I think it is equivalent to not grouping by "Amount_A" -- and grouping is helpful for the ordering.